src/resolvers/raw.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+const errCode = require('err-code')
+const extractDataFromBlock = require('../utils/extract-data-from-block')
+const validateOffsetAndLength = require('../utils/validate-offset-and-length')
+
+const rawContent = (node) => {
+  return function * (options = {}) {
+    const {
+      offset,
+      length
+    } = validateOffsetAndLength(node.length, options.offset, options.length)
+
+    yield extractDataFromBlock(node, 0, offset, offset + length)
```

Comment:
What is zero?

Reply:
The index of the first byte of the node in the stream.  Since the stream of a raw node is only as long as the node, it'll always start at 0.